pom.xml

```diff
@@ -49,9 +49,9 @@
         <picocli.version>4.7.6</picocli.version>
         <netty.version>4.1.119.Final</netty.version>
         <netty.io_uring.version>0.0.26.Final</netty.io_uring.version>
-        <netty.epoll.classifier>linux-x86_64</netty.epoll.classifier>
-        <netty.io_uring.classifier>linux-x86_64</netty.io_uring.classifier>
-        <netty.kqueue.classifier>osx-x86_64</netty.kqueue.classifier>
+        <netty.epoll.classifier>linux-${os.detected.arch}</netty.epoll.classifier>
+        <netty.io_uring.classifier>linux-${os.detected.arch}</netty.io_uring.classifier>
+        <netty.kqueue.classifier>osx-${os.detected.arch}</netty.kqueue.classifier>
```

Comment:
https://github.com/tisonkun/os-detector/ says the attribute is named `osdetector.arch`. But I guess you tested this and it works so :man_shrugging:

Reply:
Yes, that was very annoying 20 mins while I tried to work out why nothing worked. 

Its probably worth mentioning that I used `os-${arch}` instead of `${classifier}` as there is no `linux-x86_64` version of kqueue or an `osx` version of epoll. So this way the artefacts stay resolvable when building on the other platform.